packages/owl-bot/src/core.ts

```diff
@@ -497,14 +519,9 @@ async function updatePullRequestAfterPostProcessor(
     });
   } else {
     // If the pull request is a DRAFT lock file update, close it or promote it.
-    const {data: pull} = await octokit.pulls.get({
-      owner,
-      repo,
-      pull_number: prNumber,
-    });
     if (
       pull.draft &&
```

Comment:
It seems like we want to revert this check for `draft` since we opted no to take this route when performing lock updates?

Reply:
I'd like that to fall out of scope for this pull request.  I'll remove that logic in a separate pull request.